CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ customers cannot upgrade their bootloader, its changes are recorded separately.
 - Ethereum: allow signing EIP-712 messages containing multi-line strings
 - Allow exiting the screen asking to insert the microSD card
 - HWW: add initialized status byte to _api_info response
+- SD card: delete remove sd card api
```

Comment:
confusing wording, maybe better like "Remove API to prompt removal of the microSD card"